cranelift/codegen/src/isa/unwind/systemv.rs

```diff
@@ -0,0 +1,122 @@
+//! System V ABI unwind information.
+
+use alloc::vec::Vec;
+use gimli::write::{Address, FrameDescriptionEntry};
+
+#[cfg(feature = "enable-serde")]
+use serde::{Deserialize, Serialize};
+
+type Register = u16;
+type Expression = Vec<u8>;
+
+/// Module for System V x86-64 ABI utility functions.
+#[cfg(feature = "x86")]
+pub mod x86 {
+    pub use crate::isa::x86::unwind::systemv::map_reg;
+}
```

Comment:
I think this might suffer from the same problem of #1471: when compiling for ARM, this block will be ignored and `map_reg` won't exist over in `wasmtime-environ` and `wasmtime-debug`. I guess we have the same options I mentioned in that issue:

> 1. to just get the ARM building, I could remove the feature flag on x86; then users encounter a runtime error if they try to use this FDE functionality but at least it compiles
> 2. to make map_reg functional on other platforms, we could either move map_reg out of the x86 module or create separate map_reg functions in all of the ISA modules.

Reply:
Ah, I missed that it was in wasmtime-environ and not in the cranelift build.  I like the second solution, perhaps with a default implementation that errors with a new `RegisterMappingError` indicating the ISA doesn't support gimli register mapping?

Reply:
Going to resolve this and take the conversation back to #1471.